### DISCUSSION TITLE
AttributeError: ‘str‘ object has no attribute ‘decode

### DISCUSSION THREAD

**Discussion by zhangc927** [DoubleML/doubleml-for-py#165] [General]
Thank you for your great work. When I USE DML, there is a coding error in the program. Could you please tell me how to solve it？

code：
dml_plr_obj = dml.DoubleMLIRM(dml_data,
                              ml_g=LassoCV(cv=5), 
                              ml_m=LogisticRegression()
                              )

result=dml_plr_obj.fit().summary

error：
AttributeError: ‘str‘ object has no attribute ‘decode

**Comment by PhilippBach** [comment]
Dear @zhangc927 , 

thanks for opening the discussion and your interest in DoubleML... Unfortunately, I cannot replicate the error based on your code example.

It would be great if you could add information on how you created the data backend `dml_data`. Please also include the modules that you import via `import`.

Thanks, 

Philipp
